### ISSUE TITLE
Unable to remove X1 Carbon printer from Presets

### ISSUE THREAD

**Issue by MarkDerbyshire** [bambulab/BambuStudio#2911]
### Bambu Studio Version

1.7.7.89

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11 Home, Windows 10 Home

### Printer

Bambu Lab P1S

### How to reproduce

1. Go to Printer
2. Select Printer Dropdown
3. X1 Carbon is still listed even though it is deselected in Printer selection popup
![image](https://github.com/bambulab/BambuStudio/assets/20886024/1e941411-452b-4483-9c37-60697f19da3a)
![image](https://github.com/bambulab/BambuStudio/assets/20886024/f0e09550-191a-4e3a-976e-3220d8403345)

### Actual results

Bambu Labs X1 carbon still shows in dropdown

### Expected results

Bambu Lab X1 carbon not to show in dropdown

### Project file & Debug log uploads

N/A

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by bambu123** [comment]
Please provide application logs and configuration files (BambuStudio.conf).

How To:
https://wiki.bambulab.com/en/x1/troubleshooting/bambu-studio-log

**Comment by nickr1957** [comment]
Any update? I want to deregister my P1P and reregister it as P1S now I have firmware that can say it has ben upgraded...

**Comment by bambu123** [comment]
> Any update? I want to deregister my P1P and reregister it as P1S now I have firmware that can say it has ben upgraded...

We didn't reproduce it. Please provide application logs and configuration files (BambuStudio.conf).

**Comment by An21V1rus** [comment]
I am having this exact same problem on version 1.9.2.57.

![image](https://github.com/bambulab/BambuStudio/assets/63268257/8930b83d-feed-4453-9529-5a59351c7752)

![image](https://github.com/bambulab/BambuStudio/assets/63268257/39c493cd-a9e0-4edb-9c1b-ae92eb1b66ff)

[Config and Logs.zip](https://github.com/user-attachments/files/15842508/Config.and.Logs.zip)

**Comment by An21V1rus** [comment]
I actually think I figured out what is happening for me, as I'm fairly new to Bambu Studio itself. 

When I load a 3MF from MakerWorld, it seems to set the project up using whatever the creator had set for the file. I have to close Bambu Studio and open a blank project to get the X1C printer to disappear. 

Not sure if that's intended functionality, but I could see why it would be.

**Comment by DanBao-Bambu** [comment]
May I ask if the opened file contains GCode? If it does, the preset will be displayed, even if it is not checked in the wizard.

**Comment by An21V1rus** [comment]
Yes, I believe that is the case. I've been seeing it off and on depending on the randomness of where I'm getting my files (MakerWorld, Thingiverse, etc.)

**Comment by cforsyth68** [comment]
I have the exact same problem, and it's the end of 2025, a year and a half later, no resolution?
